cmd/build.go

```diff
@@ -385,9 +391,14 @@ func buildNormal(targetOS string, vmArguments []string) {
 
 	checkFlutterChannel()
 
-	var trackWidgetCreation string
+	var (
+		trackWidgetCreation string
+		verbose             string
+	)
+
 	if buildDebug {
 		trackWidgetCreation = "--track-widget-creation"
+		verbose = "-v"
```

Comment:
This is to verbose again. If you want to enable it you need to add a flag to hover. But that Should be in another PR, but actually could be used in a lot of cases.

Reply:
I was thinking about that. I didn't actually mean for it to be committed in this pr.